guides/advanced_guides/tokenization.md

```diff
@@ -21,24 +21,26 @@ We can break down the tokenization process like so:
 
 ## Language Support
 
-Currently, the MeiliSearch tokenizer is optimized for four languages:
+Currently, MeiliSearch offers full support for two groups of languages:
 
-- **English**  🇬🇧
+- **English** 🇬🇧 and **any other language that uses the Latin alphabet**
 - **Chinese** 🇨🇳
-- **Japanese** 🇯🇵
-- **Korean** 🇰🇷
 
-It also supports any language that uses the Latin alphabet. **Results may vary in languages with long compound words, such as German.**
+This is due to how our [tokenizer](/guides/advanced_guides/tokenization.md#tokenization) is optimized. While we can't promise high-quality results for languages outside of these two groups, we find that **MeiliSearch works effectively in most languages**.
```

Comment:
I'm not comfortable with this section, it make it sound like meili may not work well on most languages. I would turn it to our advantage and say that we have multi-lingual support, and that chinese is supported through jieba.

It's better to have user come to us with concrete error than people that get turned down from trying meili so we don't know how to improve their experience